types/slonik/index.d.ts

```diff
@@ -265,11 +267,18 @@ export interface QueryContextType {
      * Unique query ID
      */
     queryId: QueryIdType;
-    stackTrace: CallSiteType[] | null;
+
     /**
      * `process.hrtime.bigint()` for when query was received.
      */
     queryInputTime: number;
+
+    /** Object used by interceptors to assign interceptor-specific, query-specific context. */
+    // tslint:disable-next-line ban-types
```

Comment:
Why not use `object`?

Reply:
I am not familiar with TS enough to comment, but pretty sure it should be `any`.

It is basically an object that interceptors use to assign any arbitrary properties associated with the query.

Reply:
I've updated the `sandbox` type with `Record<string, any>`, which provides slightly better type safety compared to `any`